gsa/src/web/pages/cves/__tests__/detailspage.js

```diff
@@ -153,38 +78,45 @@ beforeEach(() => {
 });
 
 describe('CVE Detailspage tests', () => {
-  test('should render full Detailspage', () => {
+  test('should render full Detailspage', async () => {
     const getCve = jest.fn().mockResolvedValue({
-      data: entity_v2,
+      data: cve,
     });
 
     const gmp = {
-      cve: {
+      [entityType]: {
```

Comment:
I don't think we need to change this to [entityType]. That was a weird thing I did for the scanconfigs and I'm not even sure it is necessary there. It should work fine with cve just like it was before.

Reply:
I have `const cve = Cve.fromObject(cveEntity);` so if I revert that, I would have to rename that variable ...

Reply:
Did you try it? I don't think you need to rename it, one is a variable and the other one is part of an object, there should not be a problem there.

Reply:
Yeah I just missed the : ...